db/index.js

```diff
@@ -0,0 +1,32 @@
+var faker = require('faker');
+var mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost/mtnOlympus');
+
+var listingSchema = mongoose.Schema({
+  propId: Number,
+  images: [{
+    url: String,
+    description: String}]
+});
+
+var Listing = mongoose.model('Listing', listingSchema);
+
+var getListing = (index, callback) => {
+  console.log('hell from getListing')
```

Comment:
I actually like this message as-is, but maybe fix hell -> hello in the code clean up :)